README.md

```diff
@@ -15,3 +15,38 @@ Run `cmake` to configure the build environment and then `make` to build
 To install, use `make install`
 
     sudo make install
+
+## Writing contract files
+Elementary OS's Pantheon's file manager supports adding options to the context menu by the way of Contract files.
+
+Contract files can be made system wide available by adding them to:
+`/usr/share/contractor`
+or for specific users by adding them to:
+`~/.local/share/contractor`
+
+Simple .contract file example:
+```
+[Contractor Entry]
+Name=Mount
+MimeType=application/x-cd-image;application/x-raw-disk-image
+Exec=gnome-disk-image-mounter %f
+```
+
+- `Name`: Text displayed in the right click context menu.
+- `MimeType`: Mimetype(s) of files it should be shown for.
+- `Exec`: Command to execute. [More info](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#exec-variables).
+
+Tips:
+- Use `pkexec` to ask for root permissions. Example: `Exec=pkexec chmod +x %U`
+- To execute more complex scripts, put the script in a .sh file and call it.
```

Comment:
Probably shouldn't encourage the use of scripts here, especially not with elevated permissions. Contracts should expose interfaces of proper apps with policy files